web/text_layer_builder.js

```diff
@@ -321,9 +323,14 @@ var TextLayerBuilder = (function TextLayerBuilderClosure() {
     _bindMouse: function TextLayerBuilder_bindMouse() {
       var div = this.textLayerDiv;
       var self = this;
+      var expandDivsTimer = null;
       div.addEventListener('mousedown', function (e) {
         if (self.enhanceTextSelection && self.textLayerRenderTask) {
           self.textLayerRenderTask.expandTextDivs(true);
+          if (expandDivsTimer) {
+            clearTimeout(expandDivsTimer);
+            expandDivsTimer = null;
+          }
```

Comment:
Given that this only appears to be an issue in non-Firefox browsers, I think that we should thus enclose this code in appropriate preprocessor tags; i.e. something like this:
```javascript
//#if !(MOZCENTRAL || FIREFOX)
          if (expandDivsTimer) {
            clearTimeout(expandDivsTimer);
            expandDivsTimer = null;
          }
//#endif
```